casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -133,7 +148,12 @@ export function CountryPicker({
                     </UILabel>
                     <View style={styles.sideHeaderView} />
                 </View>
-                <UISearchBar returnKeyType="done" value={search} onChangeText={setSearch} />
+                <UISearchBar
+                    returnKeyType="done"
+                    value={search}
+                    onChangeText={setSearch}
+                    onFocusChanged={setFocused}
```

Comment:
Not sure that you need to create a new prop though, also not sure that you should use state there, maybe just pass `onFocus` and `onBlur` and call methods there?

Reply:
@savelichalex onBlur also does not work on my android.

Reply:
@savelichalex okay it work again

Reply:
@savelichalex sometimes